src/collections/Form/Form.js

```diff
@@ -3,17 +3,54 @@ import $ from 'jquery'
 import React, { Component, PropTypes } from 'react'
 import classNames from 'classnames'
 import META from '../../utils/Meta'
+import { getPluginProps, getComponentProps } from '../../utils/propUtils'
+import * as deprecate from '../../utils/deprecate'
+
+const pluginPropTypes = {
+  // form settings
+  keyboardShortcuts: PropTypes.bool,
+  on: PropTypes.oneOf([
+    'blur',
+    'change',
+    'submit',
+  ]),
+  revalidate: PropTypes.bool,
+  delay: PropTypes.bool,
+  inline: PropTypes.bool,
+  transition: PropTypes.string,
+  duration: PropTypes.number,
+  // callbacks
+  onValid: PropTypes.func,
+  onInvalid: PropTypes.func,
+  onSuccess: PropTypes.func,
+  onFailure: PropTypes.func,
+  fields: PropTypes.object,
+}
```

Comment:
This is the pattern we are using for separating plugin props from component props.  There is a utility to pick `this.props` for only plugin props, useful when calling refresh on the plugin. There's also a util for omitting plugin props, useful when spreading user props or doing other component prop checks.